README.md

```diff
@@ -275,12 +276,12 @@ Curable errors contain additional information about the disallowed sequence and
 
 Disallowed name errors are not considered curable because it may be challenging to suggest a specific normalization suggestion that might resolve the problem.
 
-| `DisallowedSequenceType` | General info |
-| ------------------------- | ------------ |
-| `EMPTY_NAME`   | No valid characters in name |
-| `NSM_REPEATED` | Contains a repeated non-spacing mark |
-| `NSM_TOO_MANY` | Contains too many consecutive non-spacing marks |
-| `CONF_WHOLE` | Contains visually confusing characters from {script1} and {script2} scripts |
+| `DisallowedSequenceType` | General info | Explanation |
+| ------------------------- | ------------ | ------------------------ |
+| `EMPTY_NAME` | No valid characters in name | Input was non-empty but all characters were ignorable |
```

Comment:
@lightwalker-eth interesting idea with the hollow label.
It actually made me realize my mistake with the updated `EMPTY_NAME` (hollow name) error. I made this error non-curable but since the empty string is allowed, we can cure `EMPTY_NAME` by returning the empty string.
This should make the `EMPTY_NAME` error obsolete.
We have the old `EMPTY_LABEL` curable error. It occurs for empty string labels and for ignorable hollow labels.
The cure is to remove the label. My mistake was that I thought we needed a separate non-curable error for the fully ignorable name case. But since we can cure the ignorable name with the empty string, `EMPTY_LABEL` is the correct curable error in this case.
Let me know if this makes sense.

Reply:
So basically I want to keep the curable `EMPTY_LABEL` and remove `EMPTY_NAME`.